presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1739,6 +1758,12 @@ public String getQueryId()
             return queryId;
         }
 
+        @JsonProperty
```

Comment:
Note that JSON serialization was removed in 3c8913761176675a4fc85246c64d2369faae811c, so make sure to remove when rebasing